Makefile

```diff
@@ -169,6 +169,10 @@ yamllint:
 pylint:
 	pylint $(shell find $(ROOT_DIR)  -name '*.py') -E
 
+.PHONY: shellcheck
+shellcheck:
+	shellcheck --severity=warning --format=gcc --shell=bash $(shell find . -type f -name '*.sh') build/reset  build/sed-in-place
```

Comment:
I don't see an equivalent to `ignore: olm`. This might be a github action only option, so it might come down to adjusting the `find` command.

Reply:
I had noticed this point and  before raising this PR, I have searched for `ignore` and `olm` in the context of shellcheck with no helpful results.

I can only assume that it is used as some kind of input file search pattern by the action.

So it could indeed be translated to an adjustment of the `find` command. There are, however, no `*.sh` files matching `olm`in the rook repo. I was therefore unable to come up with a reasonable adjustment ...